src/modules/core/producer_colour.c

```diff
@@ -155,7 +170,6 @@ static int producer_get_image( mlt_frame frame, uint8_t **buffer, mlt_image_form
 			memset(p, 0, size);
 			break;
 		default:
```

Comment:
Suggestion: Explicitly set this case to mlt_image_rgb24a and either remove the default case or turn it into an error case.